/gps_to_odom_node:
  ros__parameters:
    topics:
      gps_data: "/emlid/navsatfix"
      imu: "/mti680G/imu/data"
      output_odom: "/odom/gps"
    calibration:
      count: 10
    correction:
      position_rotation: -2.515 # rad
      yaw_rotation: -1.600 # rad
